### DISCUSSION TITLE
Load custom CSS in editor and embeds?

### DISCUSSION THREAD

**Discussion by addyzone** [live-codes/livecodes#411] [Q&A] [answered]
Hi! Loving LiveCodes, building out a snippets section in my blog using the embeds currently. I would like to style it up a bit, how would 
be best to add some custom CSS to both the editor and embeds? 

I can see a "Themes" page in the docs which I assume may be the feature I'm looking for, but the page is blank.

Thank you!

**Comment by addyzone** [comment]
Answered in issue :)

**Comment by hatemhosny** [comment, marked as answer]
#414 

#415
